applications/nvidia_nim/nvidia_nim_nvclip/README.md

```diff
@@ -0,0 +1,73 @@
+# NVIDIA NV-CLIP
+
+NV-CLIP is a multimodal embeddings model for image and text and this is a sample application that shows how to use the OpenAI SDK with NVIDIA Inference Microservice (NIM). Whether you are using a NIM from [build.nvidia.com/](https://build.nvidia.com/) or a self-hosted NIM, this sample application will work for both.
+
+### Quick Start
+
+1. Add API key in `nvidia_nim.yaml`
+2. `./dev_container build_and_run nvidia_nim_nvclip`
+
+## Configuring the sample application
+
+Use the `nvidia_nim.yaml` configuration file to configure the sample application:
+
+### Connection Information
+
+```
+nim:
+  base_url: https://integrate.api.nvidia.com/v1
+  api_key:
+
+```
+
+`base_url`: The URL of your NIM instance. Defaults to NVIDIA hosted NIMs.
+`api_key`: Your API key to access NVIDIA hosted NIMs.
+
+## Run the sample application
+
+There are a couple of options to run the sample application:
```

Comment:
Only one option is described below, since the section `Start the Application` works in the container. For using `./run launch <app>` in host dev env, you'd have to `./run build` and pip install reqs for Python app.